python/py-mechanicalsoup/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-mechanicalsoup
+python.rootname     MechanicalSoup
+revision            0
+version             1.1.0
+
+categories-append   devel
+license             MIT
+platforms           darwin
+supported_archs     noarch
+
+description         A Python library for automating interaction with websites.
+long_description    A Python library for automating interaction with websites. \
+                    MechanicalSoup automatically stores and sends cookies, follows redirects, \
+                    and can follow links and submit forms. It doesn't do JavaScript.
```

Comment:
this could become: `long_description    ${description}. MechanicalSoup automatically.....`